src/sdam/server.ts

```diff
@@ -175,7 +175,8 @@ export class Server extends TypedEventEmitter<ServerEvents> {
       this.emit(
         Server.DESCRIPTION_RECEIVED,
         new ServerDescription(this.description.hostAddress, event.reply, {
-          roundTripTime: calculateRoundTripTime(this.description.roundTripTime, event.duration)
+          roundTripTime: this.monitor?.roundTripTime,
+          minRoundTripTime: this.monitor?.minRoundTripTime
```

Comment:
So in LB mode how does this work? I guess there's never heartbeats and rtt is "assumed" 0. is that correct?

Reply:
In load-balanced mode, we don't initialize the monitor at all, so minRoundTripTime stays by default as 0. The CSOT spec doesn't mention any special handling for load-balanced clusters.